modules/govuk/manifests/apps/account_api.pp

```diff
@@ -65,6 +81,10 @@
   $db_port = undef,
   $db_password = undef,
   $db_name = 'account-api_production',
+  $rabbitmq_hosts = ['localhost'],
+  $rabbitmq_user = 'account_api',
+  $rabbitmq_password = 'account_api',
```

Comment:
Why don't we care about these credentials being public?
Isn't this the sort of thing we usually load with a variable / secrets?